test/Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/Wire.fir

```diff
@@ -55,10 +64,16 @@ circuit Top :
   module MyView_companion :
     output io : { }
 
-    inst tap of Tap
     wire clock: Clock
-    clock <= tap.clock
     reg r: UInt<1>, clock
+
+    inst bbox1 of BlackBox_GCT
+    bbox1.a <= r
+    inst bbox2 of BlackBox_DUTAndGCT
+    bbox2.a <= r
+
+    inst tap of Tap
+    clock <= tap.clock
```

Comment:
These were rearranged to avoid churn int he test. I want to have the blackboxes early in the module so that they don't interfere with the location of the instantiated mapping module.